ajax/libs/g2plot/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "g2plot",
+  "filename": "dist/g2plot.js",
+  "version": "1.0.2",
+  "description": "An interactive and responsive charting library",
+  "homepage": "https://g2plot.antv.vision",
+  "author": "https://github.com/orgs/antvis/people",
+  "keywords": [
+    "chart",
+    "plot",
+    "antv"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/antvis/g2plot"
+  },
+  "license": "MIT",
+  "npmName": "@antv/g2plot",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.js"
```

```suggestion
        "**/*"
```
We might as well also include the source map, so we should remove the js extension from the glob.